### PR TITLE
Using newer python image (3.9.1-alpine3.13)

### PR DESCRIPTION
**Pull request by ghost**
Tested on Fedora using podman build and aquasec's Trivy  to scan final image for known vulnerabilities. No known CVEs reported by Trivy using this python image.

### CODE REVIEW

**Comment by brendongo** [comment]
Thanks for the PR @RJColeman! Looks like right now we only test semgrep on python 3.6-3.8. While I don't forsee problems running on 3.9 probably best for sanity to keep the docker container in that version range until we fix the tests (and do some extra QA). Does `3.7.10-alpine3.13` or `3.8.7-alpine3.13` report any vulnerabilities?

**Comment by chmccreery** [comment]
@cla-bot check

**Comment by chmccreery** [comment]
@brendongo yes, she sent an email about it and we asked her for this PR as a result. I will forward you the email.

File: Dockerfile [hunk 1]
```diff
@@ -45,7 +45,7 @@ RUN ./semgrep-core/_build/install/default/bin/semgrep-core -version
 # We change container, bringing only the 'semgrep-core' binary with us.
 #
 
-FROM python:3.7.7-alpine3.11
+FROM python:3.9.1-alpine3.13
```
Comment (brendongo) [inline Dockerfile:48]:
```suggestion
FROM python:3.7.10-alpine3.13
```
Assuming this doesn't report any vulns

**Comment by ghost** [comment]
Hey

I will check and test the 3.7 version of Python tonight. I’m guessing it should be fine as it appears the issues are in Alpine 3.11 image.

I’ll also shoot over the Trivy command that will provide insight if you’re not already using it.

Best
Rebecca

Ps. That was my first pull request and contribution to another project in GitHub. Thank you for that opportunity, however small.

**Comment by chmccreery** [comment]
@RJColeman thank YOU so much for taking the time to both report this vuln to us and contribute to our open-source project! Having people like you in our community makes us so much better 👏

**Comment by ghost** [comment]
Hi all

Unfortunately I have not been able to take a second look at this and likely
won't be able to in the next few weeks..  FYI though, the
python3.7.10-alpine3.13 image lists CVEs where python3.9.1-alpine3.13 does
not. To take a look (replace ~/.cache with your cache location):

$ docker run --rm -v ~/.cache aquasec/trivy -q python:3.9.1-alpine3.13

python:3.9.1-alpine3.13 (alpine 3.13.2)
=======================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)

$ docker run --rm -v ~/.cache aquasec/trivy -q python:3.7.10-alpine3.13

python:3.7.10-alpine3.13 (alpine 3.13.1)
========================================
Total: 6 (UNKNOWN: 0, LOW: 2, MEDIUM: 2, HIGH: 2, CRITICAL: 0)

[... additional detail removed here for brevity ...]

**Comment by brendongo** [comment]
Added python3.9 tests in https://github.com/returntocorp/semgrep/pull/2606

**Comment by brendongo** [comment]
Looks like python3.9 env tests worked fine. Merging this in as is. Thanks @RJColeman!

**Pull request merged**